data.sql

```diff
@@ -1,6 +1,121 @@
 /* Populate database with sample data. */
+INSERT INTO
+    animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES
+    ('Agumon', '3-2-2020', 0, '1', 10.23);
 
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '3-2-2020', 0, '1', 10.23);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '15-11-2018', 2, '1', 8);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '7-1-2021', 1, '0', 15.04);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '12-5-2017', 5, '1', 11);
\ No newline at end of file
+INSERT INTO
+    animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES
+    ('Gabumon', '15-11-2018', 2, '1', 8);
+
+INSERT INTO
+    animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES
+    ('Pikachu', '7-1-2021', 1, '0', 15.04);
+
+INSERT INTO
+    animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES
+    ('Devimon', '12-5-2017', 5, '1', 11);

```

Comment:
- [ ] Let's be consistent with the right data asked to be entered. It turns out you added more. This is so because this milestone has skills that need to be tested, and that is why instructions were given with precise data to be entered. Kindly stick to this 👍 

- Starts from `Charmander` to `Ditto`

### Requested data:
![image](https://github.com/Mike47ip/vet-clinic-database/assets/50990925/3715faa2-e7e4-48a9-a486-862b3e93731d)

Reply:
Addressed